src/components/FeatureViews/Availability/CreateOOOForm.js

```diff
@@ -194,38 +195,62 @@ export default function CreateOOOForm() {
                 >
                     <Grid item style={{ padding: '5%' }}>
                         {!submitted && (
-                            <Button
-                                style={{
-                                    backgroundColor: 'white',
-                                    color: 'black',
-                                    width: '150px',
-                                }}
-                                onClick={() =>
-                                    SubmitNoticeChild.current.handleClearForm()
-                                }
-                                color="primary"
-                                variant="outlined"
-                                data-cy="clear-OOO-button"
-                            >
-                                Clear
-                            </Button>
+                            <ResponsiveButton
```

Comment:
Be sure to get rid of these comments in this responsiveButton as well as starting at line 214